config/tsconfig.emit-declarations.json

```
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.workspace.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true,
+    "noEmit": false
+  }
```

worth adding our `outDir` and `rootDir` here?

```suggestion
  "compilerOptions": {
    "declaration": true,
    "declarationMap": true,
    "emitDeclarationOnly": true,
    "noEmit": false
  }
```

so simplify our packages extending it